x/qgb/overview.md

```diff
@@ -0,0 +1,118 @@
+- The power threshold: the threshold defining 2/3rds of the validator set.
+- The validator set hash: The keccak256 hash of the validator set which is calculated using the [`computeValidatorSetHash()`](https://github.com/celestiaorg/quantum-gravity-bridge/blob/3cef3f5dfd37c3086fa40a6324f144595726dc16/src/QuantumGravityBridge.sol#L131-L135) method.
+
+#### Data commitment digest
+
+A data commitment digest is created inside the [`domainSeparateDataRootTupleRoot()`](https://github.com/celestiaorg/quantum-gravity-bridge/blob/3cef3f5dfd37c3086fa40a6324f144595726dc16/src/QuantumGravityBridge.sol#L156-L170) method. It is the `keccak256` hash of the concatenation of the following fields:
+
+- `DATA_ROOT_TUPLE_ROOT_DOMAIN_SEPARATOR`: which is defined as a [constant](https://github.com/celestiaorg/quantum-gravity-bridge/blob/3cef3f5dfd37c3086fa40a6324f144595726dc16/src/Constants.sol#L8-L10).
+- The data commitment nonce: the [universal nonce](https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-004-qgb-relayer-security.md#decision) of the attestation representing that data commitment.
+- The data root tuple root: which is the commitment over the set of blocks defined [above](#data-commitments).
+
+### Signatures
+
+The signature scheme used for signing the above hashes follow the [EIP-191](https://eips.ethereum.org/EIPS/eip-191) signing standard. It uses the `ECDSA` algorithm with the `secp256k1` curve. So, the orchestrator uses the keystore to [generate]((https://github.com/celestiaorg/orchestrator-relayer/blob/09ebfdc312c0d9e08856fb98cfd089e956ab7f3a/evm/ethereum_signature.go#L18-L28)) these signatures.
+
+The output signature is in the `[R || S || V]` format where `V` is `0` or `1`. This is defined in the QGB smart contract using the [Signature](https://github.com/celestiaorg/quantum-gravity-bridge/blob/3cef3f5dfd37c3086fa40a6324f144595726dc16/src/QuantumGravityBridge.sol#L17-L21) struct.
+
+These signatures are then verified in the smart contract using the [`verifySig()`](https://github.com/celestiaorg/quantum-gravity-bridge/blob/3cef3f5dfd37c3086fa40a6324f144595726dc16/src/QuantumGravityBridge.sol#L124-L129) method.
+
+## Security assumptions
+
+The QGB security relies on Celestia validator set security, which assumes that the majority of the participating validator aren't Byzantine and behave honestly. This assumption ensures that the honest nodes have control over the consensus outcome and can overpower any malicious behavior from the Byzantine nodes.
+
+So, if more than 1/3rd of the validator set stops running their orchestrators,then the QGB halts. And, if more than 2/3rds sign invalid data, then the QGB contract will commit to invalid data and we will need to redeploy the smart contract and social slash the dishonest validators.
```

Comment:
similar to the above, besides some arbitrary changes, we should point out that social consensus might not slash the validators. its up to the communitiy to determine if it was a bug or whatever. its a super minor thing, but this was also reworded to avoid the usage of "we" in the last sentence in attempt to be a tad bit more formal and boring
```suggestion
If more than 1/3rd of the validator set stops running their orchestrators, then the QGB halts. And, if more than 2/3rds sign invalid data, then the QGB contract will commit to invalid data. The only recovery from such a state is to revert to social consensus, potentially slashing the guilty validators and redeploying the smart contracts.
```